pkg/kubectl/cmd/util/helpers.go

```diff
@@ -225,6 +225,17 @@ func GetFlagString(cmd *cobra.Command, flag string) string {
 	return s
 }
 
+func CheckDeprecatedGlobalFlags(cmd *cobra.Command) {
```

Comment:
I thought that when I used a deprecated flag, `cobra` automatically notified me.  Is that not the case?